### ISSUE TITLE
Allure report accessibility issues

### ISSUE THREAD

**Issue by cmorten** [allure-framework/allure2#2264]
### Describe the Bug

There are a number of areas of the generated Allure report that are not considered accessible to all users under the [Web Content Accessibility Guidelines (WCAG) 2](https://www.w3.org/WAI/standards-guidelines/wcag/). This is currently impacting a range of users such as those with poor vision, colour-blindness, folks who make use of assistive technologies such as screen readers, etc.

It would be fantastic to start addressing some of these issues so Allure reports can be easily used by everyone regardless of their personal access requirements.

As a starter, here is a non-exhaustive list of serious/critical issues broken down by top level view, captured using the [Axe DevTools Chrome Extension](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd):

- Overview
  - [Elements must meet minimum color contrast ratio thresholds](https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=AxeChrome)
    - [ ] Inactive side navigation links and CTAs
    - [ ] Widget subtitles
    - [ ] Splash subtitles
    - [ ] Filled status bars
- Categories
  - [Form elements must have labels](https://dequeuniversity.com/rules/axe/4.8/label?application=AxeChrome)
    - [ ] Search input
  - [Elements must meet minimum color contrast ratio thresholds](https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=AxeChrome)
    - [ ] Inactive sorter CTAs
    - [ ] Status labels
    - [ ] Fullname body title
    - [ ] Parameter names
    - [ ] Test body step stats / info
- Graphs
  - [Scrollable region must have keyboard access](https://dequeuniversity.com/rules/axe/4.8/scrollable-region-focusable?application=AxeChrome)
    - [ ] Widgets grid
- Timeline
  - [Links must have discernible text](https://dequeuniversity.com/rules/axe/4.8/link-name?application=AxeChrome)
    - [ ] Test result links

(I've not repeated cross-cutting issues for each page).

### Steps to Reproduce

1. Generate Allure report
2. View Allure report in a browser
3. Observe visual, keyboard, and assistive technology issues

For reproducing the issues reported above:

1. Open Chrome browser
2. Install Axe DevTools extension from https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd
3. Open an Allure report in the Chrome browser
4. In Chrome DevTools navigate to the "axe DevTools" tab
5. Click "Scan ALL of my page"
6. Observe the reported results for that page

The Chrome Developer docs also have a number of runthroughs and resources for observing / testing accessibility issues, see https://developer.chrome.com/docs/devtools/accessibility/reference.

### Expected Behaviour

There should be no critical/serious accessibility issues with Allure reports, allowing all users to successfully understand a report regardless of their personal access requirements.

### What Language are you using?

JavaScript, TypeScript

### What Framework/Allure Integration you are using?

allure-playwright

### What version of Allure Integration you are using?

2.10.0

### What version of Allure Report you are using?

2.25.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cmorten** [comment]
@noomorph great stuff testing with a real AT user.

I would say that sounds consistent with the Axe analysis - the only recommendations that _could_ be impacting a screen reader are the search input label and the timeline links.

The majority of issues according to the report are for colour contrast which won’t impact no vision folks who rely on a screen reader, but will impact folks with low vision or vision deficiencies such as colour blindness who are navigating the report visually.

**Comment by baev** [comment]
By the way, we have this ticket on the radar, although it isn't yet prioritised. 

@cmorten, is it a blocker for you? Do you have some team members who are struggling with the Allure Report at the moment, or do you need to for compliance reasons?

**Comment by cmorten** [comment]
No blocker for me, so no rush on my end!

[I have a project](https://github.com/guidepup/aria-at-tests) for which I have been evaluating different reporter options, and tend to run a quick scan as a matter of course to check accessibility as part of evaluating options.

**Comment by baev** [comment]
by the most part fixed via https://github.com/allure-framework/allure2/pull/3341
